src/shared/config.ts

```diff
@@ -44,7 +45,44 @@ interface ServiceConfig {
     DEPTH: number;
     SHOW_HIDDEN: boolean;
     COLOR: boolean;
-  };
+  },
+  SHARED: {
+    PEER_ENDPOINT: string;
+    ALS_ENDPOINT: string;
+    QUOTES_ENDPOINT: string;
+    TRANSFERS_ENDPOINT: string;
+    BULK_TRANSFERS_ENDPOINT: string;
+    JWS_SIGN: boolean;
+    JWS_SIGNING_KEY: PathLike | Buffer;
+    WSO2_AUTH: {
+      staticToken: string;
+      tokenEndpoint: string;
+      clientKey: string;
```

Comment:
I noticed that `thirdparty-scheme-adapter` also added these fields (after Kevin's PR which I referenced):

```
DFSP_BACKEND_URI: {
      doc: 'host address of DFSP\'s ',
      format: '*',
      default: 'localhost:9000'
    },
    DFSP_BACKEND_SIGN_AUTHORIZATION_PATH: {
      doc: 'path use by BackendRequests.signAuthorizationRequest',
      format: '*',
      default: 'signchallenge'
    },
    DFSP_BACKEND_HTTP_SCHEME: {
      doc: 'Http scheme ',
      format: ['http', 'https'],
      default: 'http'
    },
```

@lewisdaly Are these also needed? Also I removed DFSP_ID since we already specify DFSP_ID through PARTICIPANT_ID, is that correct

Reply:
Nah we don't need those fields, and yep, `PARTICIPANT_ID` is correct.